### DISCUSSION TITLE
Improve Signed in window look

### DISCUSSION THREAD

**Discussion by Schweeeeeeeeeeeeeeee** [ppy/osu#27538] [Ideas] [answered]
![image](https://github.com/ppy/osu/assets/88291316/2c04d517-dade-4d1e-ac70-d05d28b3f12e)
I think the numbers are a bit squished together (probably because of the high rank.
I don't think it would hurt if it was moved a bit to the left would also make the padding from the border even on the left and right.
![image](https://github.com/ppy/osu/assets/88291316/93f0a240-12aa-4472-9f14-15492f132356)

**Comment by Gabixel** [comment, marked as answer]
It's already been reported that it overflows when it's a very large number (+ special per-language spacing): https://github.com/ppy/osu/issues/27296
It probably depends on how the design evolves.
